mozilla_bitbar_devicepool/configuration.py

```diff
@@ -191,8 +236,12 @@ def configure_projects(update_bitbar=False):
                 bitbar_file = bitbar_files[-1]
             else:
                 if update_bitbar:
-                    TESTDROID.upload_test_file(bitbar_project['id'],
-                                               os.path.join(FILESPATH, file_name))
+                    file_path = os.path.join(FILESPATH, file_name)
+                    if not os.path.exists(file_path):
+                        raise ConfigurationFileException("'%s' does not exist!" % file_path)
+                    else:
+                        print("File exists %s" % file_path)
```

Comment:
No need for the dangling else:. Do we need to print that the file exists?

Reply:
Nope, was for debugging. Will remove.